### DISCUSSION TITLE
Swagger on server different than localhost

### DISCUSSION THREAD

**Discussion by fpelaezt** [hagopj13/node-express-boilerplate#154] [Q&A]
This is so far the best boilerplate for a nodeJs api server, thanks so much for all the effort.

One quick question, the swagger documents works great using localhost, but when I deploy this to a server and try to use the server ip (http://server:port/api/docs/ even with NODE_ENV in development) I got a blank page. I tried modifying the swaggerDef.js file an using the server ip in the url but it didn't work. I also googled for this but still not sure how to fix it.

**Comment by Shameel123** [comment]
The route to swagger works only on development mode.
